src/api/Migrations/20230428143532_AddHydrotestCodelists.cs

```diff
@@ -0,0 +1,667 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BDMS.Migrations;
+
+public partial class AddHydrotestCodelists : Migration
+{
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.Sql(@"INSERT INTO bdms.codelist (
+                                    id_cli, geolcode,
+                                    schema_cli, code_cli,
+                                    text_cli_de, description_cli_de,
+                                    text_cli_fr, description_cli_fr,
+                                    text_cli_it, description_cli_it,
+                                    text_cli_en, description_cli_en,
+                                    order_cli, conf_cli,
+                                    default_cli, path_cli
+                                ) VALUES (
+                                    15203170, 001,
+                                    'htest101', '',
+                                    'Pump-/Injektionsversuch, konstante Rate', '',
+                                    '', '',
+                                    '', '',
+                                    '', '',
+                                    1, NULL,
+                                    False, ''
+                                );
```

Comment:
PP/REQ: Das `INSERT`-Statement lässt sich vereinfacht ausdrücken, da die Values kommagetrennt in einem Rutsch hinzugefügt werden können - die Wiederholung von `INSERT INTO` ist in diesem Fall nicht nötig.

Beispiel:
```sql
INSERT INTO bdms.codelist (
    id_cli, geolcode, schema_cli, code_cli,
    text_cli_de, description_cli_de,
    text_cli_fr, description_cli_fr,
    text_cli_it, description_cli_it,
    text_cli_en, description_cli_en,
    order_cli, conf_cli, default_cli, path_cli
) 
VALUES 
    (15203198, '005', 'htestres101', '', 'statischer Formationsdruck', '', '', '', '', '', '', 5, NULL, False, ''),
    (15203199, '006', 'htestres101', '', 'spezifischer Speicherkoeffizient', '', '', '', '', '', '', 6, NULL, False, ''),
    (15203200, '007', 'htestres101', '', 'Lugeon-Wert', '', '', '', '', '', '', 7, NULL, False, ''),
    (15203201, '008', 'htestres101', '', 'relevante Mächtigkeit', '', '', '', '', '', '', 8, NULL, False, ''),
    (15203202, '009', 'htestres101', '', 'keine Angaben', '', '', '', '', '', '', 9, NULL, False, '');

```